gen3-integration-tests/utils/gen3_admin_tasks.py

```diff
@@ -186,6 +186,37 @@ def run_gen3_job(
             )
 
 
+def fence_delete_expired_clients():
+    # Admin VM Deployments
+    if os.getenv("GEN3_INSTANCE_TYPE") == "ADMINVM_REMOTE":
+        run_gen3_job(
+            "fence-delete-expired-clients", test_env_namespace=pytest.namespace
+        )
+        job_logs = check_job_pod(
+            "fence-delete-expired-clients",
+            "gen3job",
+            test_env_namespace=pytest.namespace,
+        )
+        return job_logs["logs.txt"]
+    # Local Helm Deployments
+    elif os.getenv("GEN3_INSTANCE_TYPE") == "HELM_LOCAL":
+        # Delete expired clients
+        delete_explired_clients_cmd = "kubectl exec -i $(kubectl get pods -l app=fence -o jsonpath='{.items[0].metadata.name}') -- fence-create client-delete-expired"
+        delete_explired_client_result = subprocess.run(
+            delete_explired_clients_cmd,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            shell=True,
+            text=True,
+            timeout=10,
+        )
+        if not delete_explired_client_result.returncode == 0:
+            raise Exception(
+                f"Unable to delete expired clients. Response: {delete_explired_client_result.stderr.strip()}"
```

Comment:
maybe we shouldn't print out the error as it is

Reply:
This would only return the error, no client information would return with stderr. How would we wanna debug incase we dont have error message?

Reply:
we could provide a custom message, just worried if some kind of error would publish PII as part of the message